workitem/workitem_repository.go

```diff
@@ -17,10 +17,15 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+const orderValue = 1000
+const above = "above"
+const below = "below"
+
 // WorkItemRepository encapsulates storage & retrieval of work items
 type WorkItemRepository interface {
 	Load(ctx context.Context, ID string) (*app.WorkItem, error)
 	Save(ctx context.Context, wi app.WorkItem, modifierID uuid.UUID) (*app.WorkItem, error)
+	Reorder(ctx context.Context, direction string, targetID string, wi app.WorkItem, modifierID uuid.UUID) (*app.WorkItem, error)
```

Comment:
> @aslakknutsen Isn't a.Enum("above", "below") enough?

The enum is fine for the outward facing API side but internally, I think @aslakknutsen wants it to be a type like this:

```go
type DirectionType string
const (
  Above DirectionType = "above"
  Below DirectionType = "below"
)
```

Then you can use the `direction DirectionType` instead of the `direction string`.

@aslakknutsen correct?